addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -54,13 +54,25 @@
         max-width: 600px;
         height: auto !important;
     }
+
+    .oe_no_empty::after {
+        content: "";
+        display: table;
+        clear: both;
+    }
```

Comment:
Do something like that to avoid duplicated code
```
.question, .forum_answer {
    .oe_no_empty::after {
        content: "";
        display: table;
        clear: both;
    }
}
```